spec/puppet-syntax/manifests_spec.rb

```diff
@@ -136,6 +136,46 @@
     end
   end
 
+  describe 'app_management' do
+    context 'app_management = false (default)' do
+      it 'should fail to parse an application manifest' do
+
+        files = fixture_manifests(['test_app.pp'])
+        output,has_errors = subject.check(files)
+
+        expect(has_errors).to eq(true)
+        expect(output).to include(/error/)
+      end
+    end
+    context 'app_management = true' do
+      before(:each) {
+        PuppetSyntax.app_management = true
```

Comment:
Should there be a corresponding `after` block to reset this to false? I notice the future parser block doesn't have it, but both probably should, unless there's a global reset somewhere that I'm missing.